run-tests.ps1

```diff
@@ -64,4 +88,22 @@ Invoke-Expression $testRunnerCmd
 
 cd $PSScriptRoot
 
+
+if (0 -ne ([int]$LASTEXITCODE)) {
+  # check submodule status
+  $submoduleStatus = CheckSubmoduleStatus
+  if ([int]$submoduleStatus -eq 1) {
+    # not synced
+	Write-Host -ForegroundColor Yellow "Check if submodules are up to date. You can use 'git submodule update' to fix this";
```

Comment:
Please use spaces for indentation instead of tabs! (Like the rest of the file.)

Reply:
resolved, thanks